src/components/PhotoDetails.vue

```diff
@@ -152,7 +152,7 @@ export default {
   },
   methods: {
     onGoBackToSearchResults() {
-      this.$router.push({ name: 'browse-page', query: this.query });
```

Comment:
I see the navigation to the position is working now. However, this call to `$router.push` was added in [this PR](https://github.com/creativecommons/cccatalog-frontend/pull/142) to fix #136. Do you think there's a way to fix this issue and not call `$router.go(-1)`?

Reply:
Presently the best way I know to access savedPositions is by $router.go( ) as it triggers popstate event. I will definitely try to find a hack for this somehow.